commandCenter/centralHub/spotify/spotifyAPI.py

```diff
@@ -0,0 +1,90 @@
+import os
+import requests 
+import json
+import config                      
+from requests_oauthlib import OAuth1Session                                                                                                                                                
+from dotenv import load_dotenv, find_dotenv
+from pathlib import Path
+
+load_dotenv()
+
+#DEFINE AUTHENTICATION TOKENS
+SPOTIFY_API_KEY = os.getenv('SPOTIFY_CLIENTID')
+SPOTIFY_API_SECRET = os.getenv('SPOTIFY_CLIENT_SECRET')
+
+#DEFINE URLS
+SPOTIFY_BASE_AUTH_URL = 'https://accounts.spotify.com'
+SPOTIFY_BASE_URL = 'https://api.spotify.com'
+SPOTIFY_AUTH = '/api/token'
+SPOTIFY_ARTISTS = '/v1/artists/'
+SPOTIFY_SEARCH = '/v1/search'
+
+
+def spotify_authenticate():
+    '''
+    Obtains the authentication bearer for a user.
+    '''
+    auth_url = SPOTIFY_BASE_AUTH_URL + SPOTIFY_AUTH
+    headers = {
+    "Content-Type": "application/x-www-form-urlencoded"
+    }
+
+    data = {
+    "grant_type": "client_credentials",
+    "client_id": SPOTIFY_API_KEY,
+    "client_secret": SPOTIFY_API_SECRET
+    }
+
+    response = requests.post(auth_url, headers=headers, data=data)
+    token_info = json.loads(response.text)
+    return token_info['access_token']
+
+def get_spotify_artist(artist_name=None):
+    '''
+    Authenticate then search an artist based on an artist name. Searches for the name if id is not provided
+    '''
+
+    if artist_name == None:
```

Comment:
more common to do `is None`

Reply:
I think we can drop this check if we make `artist_name` a required argument to this function, that makes more sense to me